base/tomcat-9.0/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.dogtagpki</groupId>
+        <artifactId>pki-base</artifactId>
+        <version>${revision}</version>
+    </parent>
+
+    <artifactId>pki-tomcat-9.0</artifactId>
+    <packaging>jar</packaging>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>pki-tomcat</artifactId>
+            <version>${project.version}</version>
```

Comment:
Line 11 and 22 use `revision` and `project.version` respectively. Actually they have the same value but could they diverge in the future? (The same applies to all module `pom.xml`)

Reply:
I think this is fine, I don't think we want to start tracking versions at the module level. Life is already complicated enough with the web of versions we have already!

Reply:
I think this is just one of Maven's quirkiness that we will find as we use it more. Those properties are supposed to be identical (we want to use the same version for all modules), but for the module itself we're supposed to use `revision` (https://maven.apache.org/maven-ci-friendly.html#multi-module-setup), and for the dependencies we're supposed to use `project.version` (https://maven.apache.org/maven-ci-friendly.html#dependencies).